### PR TITLE
Remove subclassof terms

### LINKED ISSUE #7639: [Revise subclass] MONDO:0019391 fanconi anemia
**Mondo term (ID and Label)**

MONDO:0019391 fanconia anemia 

is a  subclass of 
telomere syndrome and dyostosis but it does not appear to be either of these?

**Suggested revision and reasons**

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by yshwetar**
closes #7639

Removed subclass of terms 'telomere syndrome' and 'dysotosis' for fanconi anemia

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -436528,10 +436528,8 @@ is_a: MONDO:0001713 {source="DOID:13636", source="MESH:D005199", source="Orphane
 is_a: MONDO:0003225 {source="PMID:31953710"} ! bone marrow disorder
 is_a: MONDO:0005570 ! hematologic disorder
 is_a: MONDO:0015161 {source="Orphanet:84"} ! multiple congenital anomalies/dysmorphic syndrome without intellectual disability
-is_a: MONDO:0018234 ! dysostosis
 is_a: MONDO:0019054 ! congenital limb malformation
 is_a: MONDO:0021190 {source="MESH:D005199", source="MONDO:Redundant", source="NCIT:C62505"} ! DNA repair disease
-is_a: MONDO:0100137 {source="PMID:29804726"} ! telomere syndrome
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:436534]: @yshwetar we do not only remove the SubClassOf, we need to keep track of what SubClassOf we remove, including the original source and your orcid (to indicate you reviewed it)
Please see instructions here: https://mondo.readthedocs.io/en/latest/editors-guide/g-logical-axioms/#excluded-subclassof:~:text=non%2Dlogical)%20axiom.-,Excluded%20subclassOf,%C2%B6,-Rather%20than%20removing

Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:436534]: note the addition of your orcid as a source for the "excluded subclass of" is not in the documentation yet (it is a relatively new sop)

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@yshwetar some changes are required. Please see the comment for more explanation.

**Comment by yshwetar** [comment]
Closing as this issue #7639 was addressed by the PR #7892

**Pull request closed without merging**